README.Rmd

```diff
@@ -138,6 +138,16 @@ Date variables are converted to `Date` class, and datetimes are converted to `PO
 dates$ctu05baseline[c(1,7), c("aspirin_start", "aspirin_start.date", "hiv_date", "hiv_date.datetime")]
 ```
 
+SecuTrial exports containing date variables sometimes include incomplete dates of the format mm.yyyy or yyyy. secuTrialR 
```

Comment:
Quick question: dd.yyyy or dd.mm are not possible versions? If yes, I suggest adding an "e.g." infront of the examples.

Reply:
Sounds good, I'll adapt this. Also they come as yyyy or yyyymm, not the other way around. The good news is that dd.yyyy and dd.mm seem not to be possible, according to the Secu Trial export formats manual.